client/App.tsx

```diff
@@ -1,65 +1,163 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import settings from '../public/settings.png';
 import intime from '../public/intime.png';
 import data from '../public/data.png';
 import barchart from '../public/barchart.png';
 import snippet from '../public/code-snippet.png';
+import logo from '../public/logo.png';
 import '../public/styles.css';
 
 export default function HomePage() {
+    const [loopNum, setLoopNum] = useState(0);
+    const [isDeleting, setIsDeleting] = useState(false);
+    const [text, setText] = useState('');
+    const [delta, setDelta] = useState(300 - Math.random() * 100);
+    const [index, setIndex] = useState(1);
+    const toRotate = ['GraphQL Gateway developer portal'];
```

Comment:
I think this can just save "Developer Portal"

Reply:
Ok